doc/nrf/libraries/caf/ble_state.rst

```diff
@@ -36,6 +35,8 @@ The following Kconfig options are also available for this module:
   This option is enabled by default.
 * :kconfig:option:`CONFIG_CAF_BLE_USE_LLPM` - This option enables the Low Latency Packet Mode (LLPM).
   This option is enabled by default and depends on :kconfig:option:`CONFIG_BT_CTLR_SDC_LLPM` and :kconfig:option:`CONFIG_BT_LL_SOFTDEVICE`.
+* :kconfig:option:`CONFIG_CAF_BLE_STATE_SECURITY_REQ` - The option enables automatically setting connection security to level 2 (encryption) right after Bluetooth LE connection is established.
+  If establishing the connection security level 2 fails, the device disconnects.
```

Comment:
Can we leave it as it used to be? The module disconnects only if connection security establishment failure is reported by the Bluetooth stack. It would not disconnect until Bluetooth stack reports security failure.